handbook/company/communications.md

```diff
@@ -642,6 +642,8 @@ At the end of their first two weeks of onboarding at Fleet, every new team membe
 
 Fleet prioritizes a [bias for action](https://fleetdm.com/handbook/company#ownership).  If possible, apply onboarding feedback to the handbook and issue templates in realtime, during this call.  This avoids backlogging tasks that may just get out of date before we get around to them anyway.
 
+### Performance feedback
```

Comment:
Mike: It's up to managers to give performance feedback, so its on the leadership page for managers